src/Components/TodoFilter/TodoFilter.tsx

```diff
@@ -0,0 +1,58 @@
+import classNames from 'classnames';
+import { Filter } from '../../Utils/Types/Filter';
+
+type Props = {
+  currentFilter: string,
+  onFilterChange?: (filter: string) => void
```

Comment:
```suggestion
  onFilterChange: (filter: string) => void
```
why it can be undefined